tests/integration/benchmark/nightly/g5-12xl.txt

```diff
@@ -5,7 +5,6 @@ ENGINE={vllm,lmi-dist}
 [container]
 deepjavalibrary/djl-serving:lmi-nightly
 [serving_properties]
-engine=Python
```

Comment:
Why removing it?

It doesn't hurt and make it easy to distinguish from future rust engine

Reply:
this is wrong today. For LMI-Dist, we will need MPI engine

Reply:
We shouold automatically add `option.mpi_model=true`, if not then it's a bug

Reply:
it is added